config/selectionCfg_ttCR_Legacy2016.cfg

```diff
@@ -0,0 +1,175 @@
+[selections]
+
+# pT and eta cuts are already applied at trig match level, so they are not needed here
+baseline = ((pairType == 0) || (pairType == 1) || (pairType == 2))  && nleps == 0 && nbjetscand > 1
+
+
+# 2016 deepFlavor
+# btag requirements - DeepFlavor WPs={0.0614, 0.3093, 0.7221}; // L,M,T -- Legacy 2016 DeepFlavor
+btagMM = bjet1_bID_deepFlavor > 0.3093 && bjet2_bID_deepFlavor > 0.3093 #both jets with medium btag working point
+
+
+# updated mass cut
+invEllypsMassCut = ((tauH_SVFIT_mass-129.)*(tauH_SVFIT_mass-129.))/(53.*53.) + ((bH_mass_raw-169.)*(bH_mass_raw-169.))/(145.*145.) >  1.0
+
+
+# ABCD regions used in the analysis - DeepTau
+SR       = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5                           # signal region: opposite sign, isolated taus
+SStight  = isOS == 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5                           # B region
+OSinviso = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # C region
+SSinviso = isOS == 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # D region
+
+
+# CR ttbar final category
+ttCR_invMcut = baseline , btagMM, isBoosted != 1 , invEllypsMassCut, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && ((bjet1_bID_deepFlavor > 0.3093) || (bjet2_bID_deepFlavor > 0.3093)))
+
+
+#########################################################################
+#########################################################################
+## weights to be applied for a certain selections when calling Fill()
+## multiple weights are passed as list and are multiplied together
+## NOTE: no weight is applied for data (the simple Fill() is used)
+[selectionWeights]
+baseline = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT
+btagMM   = bTagweightM
```

Comment:
I agree with these, just remember that if you want to use btagweightreshape bla bla it goes in baseline!

Reply:
in this moment btagweightreshape I'm not considering it, but later I will do some check on it too